pom.xml

```diff
@@ -62,6 +63,12 @@
     <logback.version>1.2.3</logback.version>
     <pax-exam.version>4.12.0</pax-exam.version>
     <simulacron.version>0.8.4</simulacron.version>
+    <jsr353-api.version>1.0</jsr353-api.version>
+    <jersey.version>2.23.1</jersey.version>
+    <hk2.version>2.4.0-b34</hk2.version>
+    <jax-rs.version>2.0.1</jax-rs.version>
+    <jsr353-ri.version>1.1.4</jsr353-ri.version>
+    <jackson.version>2.9.8</jackson.version>
```

Comment:
Duplicate property: there is already a similar declaration above: `<jackson.version>2.9.5</jackson.version>`.